lhotse/dataset/speech_synthesis.py

```diff
@@ -19,26 +19,35 @@ class SpeechSynthesisDataset(torch.utils.data.Dataset):
         {
             'audio': (B x NumSamples) float tensor
             'features': (B x NumFrames x NumFeatures) float tensor
-            'tokens': (B x NumTokens) long tensor
+            'tokens': (B x NumTokens) long tensor  # when return_tokens=True
             'audio_lens': (B, ) int tensor
             'features_lens': (B, ) int tensor
-            'tokens_lens': (B, ) int tensor
+            'tokens_lens': (B, ) int tensor  # when return_tokens=True
+            'text': List[str] of len B  # when return_tokens=False
+            'cut': List of Cuts  # when return_cuts=True
         }
     """
 
     def __init__(
         self,
-        cuts: CutSet,
+        return_cuts: bool = False,
```

Comment:
This change is breaking, can you move `return_cuts` to some position towards the end of parameter list?

Reply:
OK.